src/main/resources/db/changelog/alert/5.3.0/permissions-matrix.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="bmandel" id="2019-11-18-14-48-50-829">
+        <renameColumn tableName="PERMISSION_MATRIX" oldColumnName="ACCESS_OPERATION_ID" newColumnName="OPERATIONS" schemaName="ALERT"/>
+        <modifyDataType tableName="PERMISSION_MATRIX" columnName="OPERATIONS" schemaName="ALERT" newDataType="int"/>
+        <dropTable tableName="ACCESS_OPERATIONS" schemaName="ALERT"/>
+        <delete tableName="PERMISSION_MATRIX" schemaName="ALERT"/>
+
+        <!-- Admin permissions -->
+
+        <insert tableName="PERMISSION_MATRIX" schemaName="ALERT">
+            <column name="ROLE_ID" valueComputed="SELECT ID FROM ALERT.ROLES WHERE ROLENAME = 'ALERT_ADMIN'"/>
```

Comment:
Do we have a stored procedure to get the role id?

Reply:
GET_ID_FOR_USER_ROLE

Reply:
I would also use `GET_ID_FOR_REGISTERED_DESCRIPTOR_NAME('provider_blackduck)` for the DESCRIPTOR_ID column also.  Also use `GET_ID_FOR_CONFIG_CONTEXT` for the CONFIG context